examples/builtins/validatingwebhook.go

```diff
@@ -34,7 +34,7 @@ type podValidator struct {
 	decoder *admission.Decoder
 }
 
-// podValidator admits a pod iff a specific annotation exists.
+// podValidator admits a pod if a specific annotation exists.
```

Comment:
I don't know if this is a typo. "iff" is an abbreviation for "if and only if". Looks intended in this context.

Reply:
oh i was not aware of this :)